src/game-gui.cc

```diff
@@ -2402,8 +2402,12 @@ game::refresh_widgets()
         this->wdg_advanced->add();
         if (this->state.advanced_mode) {
             this->wdg_advanced->s[0] = gui_spritesheet::get_sprite(S_ADVUP);
+            this->wdg_layervis->add();
```

Comment:
This makes the level description button show up in the sandbox which is undesirable. Also neither of these should be hidden in the advanced mode dropdown - it seems like what you'd want to do is to hide the advanced mode show/hide button in the paused puzzle state since it doesn't do anything there.